sandbox/modules/identity/src/internalClusterTest/java/org/opensearch/identity/HttpSmokeTestCaseWithIdentity.java

```diff
@@ -38,6 +42,13 @@
 // TODO not sure why ThreadLeakScope.NONE is required
 @ThreadLeakScope(ThreadLeakScope.Scope.NONE)
```

Comment:
I know this is old but this is overriding thread leak behavior to do no checks. I assume that if NONE is required it is detecting either a SUITE level leak (the whole suite may not have leaks but the individual tests can) or a TEST level leak (no test may have a leak). 

I do not know if we need to be concerned about this. I am not sure where we would be swapping threads in this test case but imagine it is probably when it  starts the second node or when it goes to check the cluster health.